From: hyoyeong <164199371@qq.com>
Date: Thu, 27 Aug 2026 02:33:03 +0000
Subject: =?utf-8?b?VHJhbnNsYXRlZCB1c2luZyBXZWJsYXRlICjmnJ3pspzor60gKGtvX0tQ?=
 =?utf-8?b?KSk=?=

Currently translated at 100.0% (139 of 139 strings)

Translation: openkylin-huanghe/kylin-os-manager-fault-diagnosis
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager-fault-diagnosis/ko_KP/
---
 plugins/fault_diagnosis/translations/fault-diagnosis_ko_KP.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/fault_diagnosis/translations/fault-diagnosis_ko_KP.ts b/plugins/fault_diagnosis/translations/fault-diagnosis_ko_KP.ts
index 0ba7b04..d16cdaa 100644
--- a/plugins/fault_diagnosis/translations/fault-diagnosis_ko_KP.ts
+++ b/plugins/fault_diagnosis/translations/fault-diagnosis_ko_KP.ts
@@ -386,7 +386,7 @@
     <message>
         <location filename="../home_page_widget.cpp" line="193"/>
         <source>Application detection</source>
-        <translation>응용프로그람 검사</translation>
+        <translation>응용 프로그람 검사</translation>
     </message>
     <message>
         <location filename="../home_page_widget.cpp" line="194"/>
@@ -491,7 +491,7 @@
     <message>
         <location filename="../mainwindow.cpp" line="250"/>
         <source>Application detection</source>
-        <translation>응용프로그람 검사</translation>
+        <translation>응용 프로그람 검사</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="306"/>
